### PR TITLE
Update LocalVMSetup.md

### PR DESCRIPTION
**Pull request by hessio**
I updated the commands because the commands at the given link do not help the user to get setup

### CODE REVIEW

**Comment by mukul-seagate11** [comment]
> I updated the commands because the commands at the given link do not help the user to get setup

Can you clarify in more details with logs or screenshot as the commands mentioned are generic command as per platform,

**Comment by VenkyOS** [comment]
@hessio - If you are using google chrome as the browser, you can use the below mentioned extension.

[DCO_Extension](https://chrome.google.com/webstore/detail/dco-github-ui/onhgmjhnaeipfgacbglaphlmllkpoijo?hl=en)

File: doc/LocalVMSetup.md [hunk 1]
```diff
@@ -16,8 +16,18 @@ You can download CentOS 7.8.2003 ISO file from http://www.centos.org, or from an
 ## 3. Install a fresh CentOS 7.8.2003
 Automatic install or manual install. Basic installation is OK. When you build CORTX from source in next steps, dependant packages will be resolved and installed.
 
-## 4. You may need to add EPEL repo.
-Please refer to: https://fedoraproject.org/wiki/EPEL
+## 4. Run the following commands.
```
Comment (andriytk) [inline doc/LocalVMSetup.md:19]: Please explain what these commands do. Do they add EPEL (like it was mentioned before)? If so, please mention it.

File: doc/LocalVMSetup.md [hunk 2]
```diff
@@ -16,8 +16,18 @@ You can download CentOS 7.8.2003 ISO file from http://www.centos.org, or from an
...
+
+$ sudo yum install epel-release
```
Comment (andriytk) [inline doc/LocalVMSetup.md:21]: Use [Markdown formatting](https://arcticicestudio.github.io/styleguide-markdown/rules/code.html#no-shell-code-dollar-sign) for commands:

```bash
sudo yum install epel-release
```

File: doc/LocalVMSetup.md [hunk 3]
```diff
@@ -16,8 +16,18 @@ You can download CentOS 7.8.2003 ISO file from http://www.centos.org, or from an
...
+
+$ sudo yum repolist
+
+$ sudo yum install ansible
+
+$ wget https://archives.fedoraproject.org/pub/archive/epel/5/x86_64/epel-release-5-4.noarch.rpm
```
Comment (andriytk) [inline doc/LocalVMSetup.md:27]: Shouldn't it be `epel-release-latest-7` for CentOS-7 as mentioned in the link which you removed?

Comment (hessio) [reply doc/LocalVMSetup.md:27]: That didn't work for me when I used that command

File: doc/LocalVMSetup.md (hunk 1 above)
Comment (huanghua78) [reply doc/LocalVMSetup.md:19]: Yes, please keep this title unchanged:
```
You may need to add EPEL repo.
```
This tells the user what to do next.

File: doc/LocalVMSetup.md [hunk 4]
```diff
@@ -16,8 +16,18 @@ You can download CentOS 7.8.2003 ISO file from http://www.centos.org, or from an
...
+ sudo yum install epel-release
+
+ sudo yum repolist
+
+ sudo yum install ansible
+
+ wget https://archives.fedoraproject.org/pub/archive/epel/5/x86_64/epel-release-5-4.noarch.rpm
+ sudo rpm –ivh epel-release-latest-5.noarch.rpm
+ sudo yum repolist
+ sudo yum install ansible
```
Comment (huanghua78) [inline doc/LocalVMSetup.md:30]: These four lines are doing the same as previous three lines? 


ps. the format of these lines are broken.

File: doc/LocalVMSetup.md (hunk 1 above)
Comment (huanghua78) [reply doc/LocalVMSetup.md:19]: Please don't change this title here.

It is referenced from "cortx-motr/doc/Quick-Start-Guide.rst"

File: doc/LocalVMSetup.md [hunk 5]
```diff
@@ -16,9 +16,16 @@ You can download CentOS 7.8.2003 ISO file from http://www.centos.org, or from an
 ## 3. Install a fresh CentOS 7.8.2003
 Automatic install or manual install. Basic installation is OK. When you build CORTX from source in next steps, dependant packages will be resolved and installed.
 
-## 4. You may need to add EPEL repo.
+## 4. Install GUI so you can setup ssh key
+yum groups install "GNOME Desktop"
+
+startx
```
Comment (huanghua78) [inline doc/LocalVMSetup.md:22]: Why is this needed?

Comment (hessio) [reply doc/LocalVMSetup.md:22]: I think this is helpful because I spent a lot of time trying to figure out how to get passed the ssh step in the quick start guide and by having a GUI it makes the ssh step a lot more straight forward

Comment (andriytk) [reply doc/LocalVMSetup.md:22]: GNOME Desktop or X11 graphics definitely should not be needed to pass the "ssh step".

Note: the VM machine is designed to work as a server node, not a desktop one.

Comment (hessio) [reply doc/LocalVMSetup.md:22]: Ok, but the way I figured out to do it was to use the Desktop GUI, how would you do it in command line?

Comment (huanghua78) [reply doc/LocalVMSetup.md:22]: Then, what is the problems in the "ssh step"?
What specific things that you have troubles to do in command line?

Comment (hessio) [reply doc/LocalVMSetup.md:22]: The problem I was having was that I wasn't able to get access, I was supposed to put in a key to my GitHub account

Comment (huanghua78) [reply doc/LocalVMSetup.md:22]: You can do this in your host OS, not in VM.

Comment (hessio) [reply doc/LocalVMSetup.md:22]: Ok, but do you hve to log into you github account then from the command line?

File: doc/LocalVMSetup.md [hunk 6]
```diff
@@ -16,9 +16,16 @@ You can download CentOS 7.8.2003 ISO file from http://www.centos.org, or from an
 ## 3. Install a fresh CentOS 7.8.2003
 Automatic install or manual install. Basic installation is OK. When you build CORTX from source in next steps, dependant packages will be resolved and installed.
 
-## 4. You may need to add EPEL repo.
+## 4. Install GUI so you can setup ssh key
```
Comment (andriytk) [inline doc/LocalVMSetup.md:19]: It's not needed (GUI for ssh).

File: doc/LocalVMSetup.md [hunk 7]
```diff
@@ -16,9 +16,16 @@ You can download CentOS 7.8.2003 ISO file from http://www.centos.org, or from an
...
+yum groups install "GNOME Desktop"
+
+startx
+
+## 5. You may need EPEl repo.
 Please refer to: https://fedoraproject.org/wiki/EPEL
 
-## 5. Now you are ready to build and test CORTX. 
+sudo yum install ansible
```
Comment (andriytk) [inline doc/LocalVMSetup.md:27]: What this command does?

Comment (hessio) [reply doc/LocalVMSetup.md:27]: installs ansible, which is needed in the prerequisites

File: doc/LocalVMSetup.md (hunk 6 above)
Comment (hessio) [reply doc/LocalVMSetup.md:19]: Okay, noted.

File: doc/LocalVMSetup.md (hunk 4 above)
Comment (hessio) [reply doc/LocalVMSetup.md]: Yes I don't think these lines are needed, you are right

File: doc/LocalVMSetup.md (hunk 7 above)
Comment (andriytk) [reply doc/LocalVMSetup.md:27]: So add a new section here, like "Motr build prerequisites" and add this command under it. Otherwise, it's not clear why it is needed.

File: doc/LocalVMSetup.md (hunk 6 above)
Comment (andriytk) [reply doc/LocalVMSetup.md:19]: I'd suggest removing this "Install GUI so you can setup ssh key" section at all.

File: doc/LocalVMSetup.md (hunk 7 above)
Comment (hessio) [reply doc/LocalVMSetup.md:27]: ok, that makes sense

**Comment by johnbent** [comment]
@andriytk and @huanghua78 , are all of your comments addressed?  If so, can you please approve?  If not, do you think this PR is worth continuing to pursue or should we drop it?

**Comment by andriytk** [comment]
@johnbent no. GUI definitely is not needed to put ssh keys. And the ssh keys configuration do not seem relevant here at all in this doc.

But I like the idea of being able to create a VM from scratch in a few simple steps. So I'd keep this doc, but clean it up and make sure all the steps work.

**Comment by huanghua78** [comment]
> @johnbent no. GUI definitely is not needed to put ssh keys. And the ssh keys configuration do not seem relevant here at all in this doc.
> 
> But I like the idea of being able to create a VM from scratch in a few simple steps. So I'd keep this doc, but clean it up and make sure all the steps work.

I agree with @andriy.

**Comment by hessio** [comment]
I think this is an unnecessary issue - @johnbent

**Comment by andriytk** [comment]
@hessio what are your plans about this PR? Please update/fix it or close it. Thanks.

**Comment by hessio** [comment]
Made this on my second day and realize it doesn't make sense to do this - therefor I am closing this PR.
